play/src/front/Components/MapEditor/PropertyEditor/AddPropertyButton.svelte

```diff
@@ -8,11 +10,31 @@
     export let disabled = false;
     export let testId = undefined;
     const dispatch = createEventDispatcher();
+
+    if (!testId) {
```

Comment:
I think it could be better to use hook of Svelte like "onMounted" to change the value

Reply:
Or force to have testId for any button created